Tests/TuistSupportTests/Utils/FileHandlerTests.swift

```diff
@@ -46,13 +46,13 @@ final class FileHandlerTests: TuistUnitTestCase {
         try FileHandler.shared.touch(from)
         let to = temporaryPath.appending(component: "to")
         
-        let count = try countItemsInRootTempDirectory(appropriateFor: to.asURL)
+//        let count = try countItemsInRootTempDirectory(appropriateFor: to.asURL)
 
         // When
         try subject.replace(to, with: from)
 
         // Then
-        XCTAssertEqual(count, try countItemsInRootTempDirectory(appropriateFor: to.asURL))
+//        XCTAssertEqual(count, try countItemsInRootTempDirectory(appropriateFor: to.asURL))
```

Comment:
Does changing `create` to `true` https://github.com/tuist/tuist/pull/663/files#diff-c1df8289f6811c9b6efbbb7a4028a49dR64 help?

```diff
fileManager.url(for: .itemReplacementDirectory,
                         in: .userDomainMask,
                         appropriateFor: url,
+                      create: true)
```

Reply:
I'll try it and we'll see what the CI has to say.

Reply:
Thanks @fortmarek!

if that doesn't work, we can disable this test and attempt to test for it in another way

For reference this test was added due to https://github.com/tuist/tuist/pull/408